src/Components/UserAuth.js

```diff
@@ -0,0 +1,62 @@
+import React, { useState } from 'react';
+import axios from 'axios';
+import { Link, useNavigate } from 'react-router-dom';
+import PropTypes from 'prop-types';
+import '../Sign-In.css';
+
+function UserAuth({ setLoggedIn }) {
```

Comment:
- Please, consider using arrow funcs according to Javascript ES6 good practices.